src/commands.ts

```diff
@@ -94,10 +95,16 @@ export function precompileFracasFile(frcDoc: vscode.TextDocument | undefined = u
         const upperRoot = frcPath.root.toUpperCase(); // ninja requires that the drive letter be uppercase
         const zoFile = `${upperRoot}${frcPath.dir.substring(upperRoot.length)}/compiled/${frcPath.name}_frc.zo`;
 
-        // invoke ninja to precompile the fracas file
-        const ninjaCmd = `${ninja} -f ./build/build_precompile.ninja ${zoFile}`;
-        console.log(ninjaCmd);
-        execShell(ninjaCmd);
+        // If the precompiled zo file exists, invoke ninja to update it
```

Comment:
Not all `.frc` files are precompiled. This check prevents spurious error message popups.

Reply:
One idea: we could modify Frake to generate another ninja file (just for this plugin) that had a list of targets, one for each `.frc` file. The dependencies (which would be empty in many cases) could list all the targets that particular `.frc` builds to.  Then the plugin could just blindly invoke ninja on that ninja file specifying the `.frc` file target (named based on the `.frc` file itself) to precompile everything.

Reply:
It could leave out targets with overly long build times (certain json files) maybe. Or maybe that would still be useful.

Reply:
I have designs on something similar to your idea. I'd like to get the plugin to where it just invokes "ninja" with no special targets or arguments and it does all the necessary dependency tracking to build not only the zo files, but everything required to live update the GDR.